apps/fabric-website/src/components/App/App.tsx

```diff
@@ -1,12 +1,14 @@
 import * as React from 'react';
-import { Fabric } from 'office-ui-fabric-react/lib/Fabric';
-import { ActionButton } from 'office-ui-fabric-react/lib/Button';
-import { Panel, PanelType } from 'office-ui-fabric-react/lib/Panel';
 import AttachedScrollUtility from '../../utilities/AttachedScrollUtility';
-import { UHFBreakPoints } from '../../utilities/WindowWidthUtility';
+import { ActionButton } from 'office-ui-fabric-react/lib/Button';
+import { AppCustomizations } from './customizations';
+import { AppCustomizationsContext } from '@uifabric/example-app-base';
```

Comment:
lines 4 and 5 are new, but tendencies required the list to be sorted as well, hence the larger change.